hilgendorf_moveit_config/config/kinematics.yaml

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Can this file be empty? I'd expect there to be a set of `KDLKinematicsPlugin` stanzas, one for each chain?